hecuba_py/tests/withQbeast/qbeast_storagedict_tests.py

```diff
@@ -70,7 +70,7 @@ def testCanBeRebuilt(self):
 
         time.sleep(1)
         filtered = filter(lambda row: row.x > 0.02 and row.x < 0.25 and row.y > 0.26 and row.y < 0.45 and row.z > 0.58 and row.z < 0.9, d.items())
-        from storage.api import getByID
+        from storage import getByID
```

Comment:
We shouldnt edit this here

Reply:
Besides, we must call it "storage.api" because of an agreement with COMPSs

```python
 from storage.api import init
    from storage.api import finish
    from storage.api import getByID
    from storage.api import TaskContext 
```